<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BusView</class>
 <widget class="QWidget" name="BusView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>381</width>
     <height>261</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>379</width>
      <height>259</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="candFramesLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Candidate Frames</string>
   </property>
  </widget>
  <widget class="QPushButton" name="transmitButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Transmit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
